src/libs/Kernel.cpp

```diff
@@ -55,13 +55,15 @@ Kernel::Kernel(){
 
     instance= this; // setup the Singleton instance of the kernel
 
+    // All streams register to the Kernel so we can broadcast messages
+    this->streams = new StreamOutputPool();
 
     // Create the default UART Serial Console interface
-    this->serial = new SerialConsole(P3_4, P3_5, 9600);
+    this->serial = new SerialConsole(P6_5, P6_4, 9600);
```

Comment:
This part of the change will break Smoothie2 hardware itself. @arthurwolf It would be best to wait and merge this PR in  once I am ready to commit in a related change that makes it easier to work with multiple boards. That way you can merge this, I will rebase/fixup my change, and then issue my related PR.